core/commonTest/src/kotlinx/serialization/internal/PrimitiveSerialDescriptorTest.kt

```diff
@@ -0,0 +1,28 @@
+package kotlinx.serialization.internal
```

Comment:
We don't usually put tests in the `internal` package, as it is not descriptive. There are already `SerialDescriptorBuilderTest` and `SerialDescriptorEqualityTest` that look like a proper fit for these test functions. Alternatively, you can just put `PrimitiveSerialDescriptorTest` near them.

Reply:
I've checked `SerialDescriptorBuilderTest` and `SerialDescriptorEqualityTest` 
For me, it seems that separate `PrimitiveSerialDescriptorTest` is better than adding test into one of these classes, so I just moved `PrimitiveSerialDescriptorTest` out of `internal` package.